library/validation/validate_core.php

```diff
@@ -76,7 +76,6 @@ function validateUsingPageRules($fileNamePath)
         //Not lbf forms use the new validation, please make sure you have the corresponding values in the list Page validation
         $use_validate_js = 1;
         require_once($GLOBALS['srcdir'] . "/validation/validation_script.js.php");
-        $new_validate = $GLOBALS['new_validate'] ? 1 : 0;
```

Comment:
What about line 102, where the $new_validate variable is used (should it be $use_validate_js instead?)?